src/model/application/command.rs

```diff
@@ -73,11 +75,23 @@ pub struct Command {
     /// Indicates whether the command is available in DMs with the app, only for globally-scoped
     /// commands. By default, commands are visible.
     #[serde(default)]
+    #[deprecated = "Use Command::contexts"]
     pub dm_permission: Option<bool>,
     /// Indicates whether the command is [age-restricted](https://discord.com/developers/docs/interactions/application-commands#agerestricted-commands),
     /// defaults to false.
     #[serde(default)]
     pub nsfw: bool,
+    /// Installation context(s) where the command is available, only for globally-scoped commands.
+    ///
+    /// Defaults to [`InstallationContext::Guild`]
+    #[cfg(feature = "unstable_discord_api")]
+    pub integration_types: Vec<InstallationContext>,
+    /// Interaction context(s) where the command can be used, only for globally-scoped commands.
+    ///
+    /// By default, all interaction context types are included.
+    #[cfg(feature = "unstable_discord_api")]
+    #[serde(default)]
+    pub contexts: Vec<InteractionContext>,
```

Comment:
tested this change just now, I was wrong :/
unless there is another nice way to deserialize `null` as an empty `Vec<T>` this should be an `Option`

```suggestion
    pub contexts: Option<Vec<InteractionContext>>,
```